Manual.md

```diff
@@ -915,6 +915,15 @@ depend on additional packages. This build style does not install
 dependencies to the root directory, and only checks if a binary package is
 available in repositories.
 
+- `nodejs` For apps made in Node.js that get built with npm or yarn
```

Comment:
I think the explanation of variables could be better formatted as a list.  Look at the python section starting on line 1504.